### PR TITLE
Translate docs/tutorials/stateless-application/expose-external-ip-address.md in Japanese

### LINKED ISSUE #15661: Translate /docs/tutorials/stateless-application/expose-external-ip-address.md in Japanese
**This is a Feature Request**

**What would you like to be added**

/docs/tutorials/stateless-application/expose-external-ip-address.md in Japanese

**Why is this needed**

No /docs/tutorials/stateless-application/expose-external-ip-address.md in Japanese.

**Comments**

Copy from content/en/docs/tutorials/stateless-application/expose-external-ip-address.md and update content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md

### PR DESCRIPTION
**Pull request by af12066**
Close #15661 

Original document: https://github.com/kubernetes/website/blob/dev-1.15-ja.1/content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

### CODE REVIEW

File: content/ja/docs/tutorials/_index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ content_template: templates/concept
 
 ## ステートレスアプリケーション
 
-* [クラスタ内のアプリケーションにアクセスするために外部IPアドレスを公開する](/docs/tutorials/stateless-application/expose-external-ip-address/)
+* [クラスタ内のアプリケーションにアクセスするために外部IPアドレスを公開する](/ja/docs/tutorials/stateless-application/expose-external-ip-address/)
```
Comment (sftim) [inline content/ja/docs/tutorials/_index.md]: This should stay as `/docs/tutorials/stateless-application/expose-external-ip-address/` I think.

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+---
+title: クラスタ内のアプリケーションにアクセスするために外部IPアドレスを公開する
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
title: クラスター内のアプリケーションにアクセスするために外部IPアドレスを公開する
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+content_template: templates/tutorial
+weight: 10
+---
+
+{{% capture overview %}}
+
+このページでは、外部IPアドレスを公開するKubernetesのServiceオブジェクトに関する作成方法を示します。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
このページでは、外部IPアドレスを公開するKubernetesのServiceオブジェクトを作成する方法を示します。
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+ * [kubectl](/docs/tasks/tools/install-kubectl/)をインストールしてください。
+
+ * Kubernetesクラスターを作成する際に、Google Kubernetes EngineやAmazon Web Servicesのようなクラウドプロバイダーを使用します。このチュートリアルでは、クラウドプロバイダーを必要とする[外部ロードバランサ](/docs/tasks/access-application-cluster/create-external-load-balancer/)を作成します。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
 * Kubernetesクラスターを作成する際に、Google Kubernetes EngineやAmazon Web Servicesのようなクラウドプロバイダーを使用します。このチュートリアルでは、クラウドプロバイダーを必要とする[外部ロードバランサー](/docs/tasks/access-application-cluster/create-external-load-balancer/)を作成します。
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+ * Kubernetes APIサーバと通信するために、`kubectl`を設定してください。手順については、各クラウドプロバイダーのドキュメントを参照してください。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
 * Kubernetes APIサーバーと通信するために、`kubectl`を設定してください。手順については、各クラウドプロバイダーのドキュメントを参照してください。
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+        curl http://<external-ip>:<port>
+
+    ここで、`<external-ip>`はServiceの外部IPアドレス(`LoadBalancer Ingress`)で、
+    `<port>`はServiceの詳細出力における`Port`となります。
+
+    minikubeを使用している場合、`minikube service my-service`を実行することでHello Worldアプリケーションをブラウザで自動的に
+    開かれます。
+
+    正常なリクエストに対するレスポンスは、helloメッセージになります:
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
    正常なリクエストに対するレスポンスは、helloメッセージにです:
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+        hello-world-2895499144-segjf ...  10.0.2.5   gke-cluster-1-default-pool-e0b8d269-cpuc
+
+1. Hello Worldアプリケーションにアクセスするために、外部IPアドレス(`LoadBalancer Ingress`)を使用します:
+
+        curl http://<external-ip>:<port>
+
+    ここで、`<external-ip>`はServiceの外部IPアドレス(`LoadBalancer Ingress`)で、
+    `<port>`はServiceの詳細出力における`Port`となります。
+
+    minikubeを使用している場合、`minikube service my-service`を実行することでHello Worldアプリケーションをブラウザで自動的に
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]: 改行はなくて良さそうです

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+        hello-world-2895499144-9m4h1 ...  10.0.0.6   gke-cluster-1-default-pool-e0b8d269-5v7a
+        hello-world-2895499144-o4z13 ...  10.0.1.7   gke-cluster-1-default-pool-e0b8d269-1afc
+        hello-world-2895499144-segjf ...  10.0.2.5   gke-cluster-1-default-pool-e0b8d269-cpuc
+
+1. Hello Worldアプリケーションにアクセスするために、外部IPアドレス(`LoadBalancer Ingress`)を使用します:
+
+        curl http://<external-ip>:<port>
+
+    ここで、`<external-ip>`はServiceの外部IPアドレス(`LoadBalancer Ingress`)で、
+    `<port>`はServiceの詳細出力における`Port`となります。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
    `<port>`はServiceの詳細出力における`Port`です。
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+        IP:             10.3.245.137
+        LoadBalancer Ingress:   104.198.205.71
+        Port:           <unset> 8080/TCP
+        NodePort:       <unset> 32377/TCP
+        Endpoints:      10.0.0.6:8080,10.0.1.6:8080,10.0.1.7:8080 + 2 more...
+        Session Affinity:   None
+        Events:         <none>
+
+    Serviceによって公開された外部IPアドレス(`LoadBalancer Ingress`)を記録しておいてください。
+    この例では、外部IPアドレスは104.198.205.71となります。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
    この例では、外部IPアドレスは104.198.205.71です。
```

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+    また、`Port`および`NodePort`の値も控えてください。
+    この例では、`Port`は8080、`NodePort`は32377となります。
```
Comment (nasa9084) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]:
```suggestion
    この例では、`Port`は8080、`NodePort`は32377です。
```

File: content/ja/docs/tutorials/_index.md (hunk 1 above)
Comment (af12066) [reply content/ja/docs/tutorials/_index.md]: Okay, I'll fix it.

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md (hunk 6 above)
Comment (af12066) [reply content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md]: https://github.com/kubernetes/website/pull/15672/commits/7c054f901740fb55251a7bbeb39b2f44b16c446b

**Comment by inductor** [comment]
lgtm

**Pull request merged**